lib/world_cup.rb

```diff
@@ -0,0 +1,25 @@
+require "pry"
+class WorldCup
+  attr_reader :year, :teams
+  def initialize(year, teams)
+    @year = year
+    @teams = teams
+  end
+
+  def active_players_by_position(position)
```

Comment:
This logic looks REALLY great!  A refactor to think about would be to split this into two methods - one to determine if the team is eliminated and then one to grab the players.  But this looks really good!